packages/mobile/src/account/Education.tsx

```diff
@@ -45,15 +49,26 @@ export default class Education extends React.Component<Props, State> {
   swiper = React.createRef<Swiper>()
 
   goBack = () => {
-    const currentStepInfo = this.props.stepInfo[this.state.step]
-    if (currentStepInfo.cancelEvent && currentStepInfo.screenName) {
-      ValoraAnalytics.track(currentStepInfo.cancelEvent, {
-        screen: currentStepInfo.screenName,
-      })
-    }
-    if (this.state.step === 0) {
+    const { step } = this.state
+    const { topic } = this.props.stepInfo[this.state.step]
+    if (step === 0) {
+      const analyticsEvent =
+        topic === EducationTopic.backup
+          ? OnboardingEvents.backup_education_cancel
+          : OnboardingEvents.celo_education_cancel
```

Comment:
Rather than assuming the education screen will be just one of the other, can this be generalized with a nav prop or something like that?

Reply:
We could. We have an `EducationTopic` enum defined in this file where the two types thus far are `backup` and `celo`. Do you think we should split this up into if statements now or wait until we add more `EducationTopic`s?